notifications/providers/telegram.py

```diff
@@ -1,8 +1,26 @@
+import telegram
+
+from ..settings import PROVIDERS_SETTINGS
 from .base import ProviderBase
 
 
 class Telegram(ProviderBase):
     code = "telegram"
 
     def _publish(self, tpl: str):
-        pass
+        """
+        Publish notifications to Telegram using a BOT
+        """
+        settings = PROVIDERS_SETTINGS.get(self.code)
```

Comment:
Maybe would be better to set a method in the parent class (ProviderBase) name `get_settings` and move this functionality there.

Reply:
Done!